### PR TITLE
Add MapCustom to Extensions logging

### PR DESCRIPTION
**Pull request by Mpdreamz**
This existed for Serilog but not Extensions Logging.

### CODE REVIEW

File: src/Elastic.Extensions.Logging.Console/LoggingBuilderExtensions.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace Elastic.Extensions.Logging.Console;
 public static class LoggingBuilderExtensions
 {
 	/// <summary> Adds ECS output to console output</summary>
-	public static ILoggingBuilder AddEcsConsole(this ILoggingBuilder builder, LogLevel stdErrorThreshold = LogLevel.Warning, Action<EcsConsoleFormatterOptions>? configure = null)
+	public static ILoggingBuilder AddEcsConsole(this ILoggingBuilder builder, Action<EcsConsoleFormatterOptions>? configure = null, LogLevel stdErrorThreshold = LogLevel.Warning)
```
Comment (stevejgordon) [inline src/Elastic.Extensions.Logging.Console/LoggingBuilderExtensions.cs:9]: Any reason to reorder here? This would break existing call sites.

**Review by stevejgordon — COMMENTED** [review]
One query, but otherwise LGTM

File: src/Elastic.Extensions.Logging.Console/LoggingBuilderExtensions.cs (hunk 1 above)
Comment (Mpdreamz) [reply src/Elastic.Extensions.Logging.Console/LoggingBuilderExtensions.cs:9]: Good catch!

**Review by stevejgordon — APPROVED** [review]
LGTM

**Pull request merged**